pages/linux/torify.md

```diff
@@ -0,0 +1,31 @@
+# torify
+
+> Route network traffic through the Tor network.
+
+- Route traffic via Tor:
+
+`torify {{command}}`
+
+- Toggle Tor in shell:
+
+`torify {{on|off}}`
+
+- Spawn a Tor-enabled shell:
+
+`torify --shell`
+
+- Check for a Tor-enabled shell:
+
+`torify show`
+
+- Specify Tor configuration file:
+
+`torify -c {{config-file}} {{command}}`
+
+- Use a specific Tor SOCKS proxy:
+
+`torify -P {{proxy}} {{command}}`
+
+- Redirect output to a file:
+
+`torify {{command}} > output.txt`
```

Comment:
```suggestion
`torify {{command}} > {{path/to/output}}`
```
IG we can convert this to a placeholder. Also, I have removed the extension as any extensions can be used.